apis/core/lib/domain/cube-projects/create.ts

```diff
@@ -1,24 +1,26 @@
 import { GraphPointer } from 'clownface'
-import { rdfs, rdf, hydra } from '@tpluscode/rdf-ns-builders'
+import { NamedNode } from 'rdf-js'
+import { rdfs, rdf } from '@tpluscode/rdf-ns-builders'
 import { cc } from '@cube-creator/core/namespace'
 import { ResourceStore } from '../../ResourceStore'
 import { cubeProject, csvMapping } from '../identifiers'
 
 interface CreateProjectCommand {
+  projectsCollection: NamedNode
   resource: GraphPointer
   store: ResourceStore
 }
 
-export async function createProject({ resource, store }: CreateProjectCommand): Promise<GraphPointer> {
+export async function createProject({ projectsCollection, resource, store }: CreateProjectCommand): Promise<GraphPointer> {
   const label = resource.out(rdfs.label).term!
 
-  const project = store.create(cubeProject(label.value))
-    .addOut(rdfs.label, label)
-    .addOut(rdf.type, [cc.CubeProject, hydra.Resource])
+  const project = await store.createMember(projectsCollection, cubeProject(label.value))
+
+  project.addOut(rdfs.label, label)
 
   if (resource.out(cc.projectSourceKind).value === 'CSV') {
     const mapping = store.create(csvMapping(project))
-      .addOut(rdf.type, [cc.CsvMapping, hydra.Resource])
```

Comment:
Why is the Resource not needed anymore?

Reply:
It became the default. Check the first point in the issue description